pkg/webhook/example_test.go

```diff
@@ -80,75 +76,3 @@ func Example() {
 		panic(err)
 	}
 }
-
-// This example creates a webhook server that can be
-// ran without a controller manager.
-//
-// Note that this assumes and requires a valid TLS
-// cert and key at the default locations
-// tls.crt and tls.key.
-func ExampleServer_StartStandalone() {
```

Comment:
What is the reason for removing this?

Reply:
StandaloneServer here makes it more difficult to support the Webhook code long term, looking at `grep.app` this code is not used on publicly available code at least. The Standalone seems it was added to use webhooks outside of a Manager, but really lots of things from the manager are then required (like scheme, or client).

Going forward, we should try to make the Manager as slim as possible, and require it explicitly

Reply:
grep.app only indexes the N most popluar repos, so it is somewhat useless when finding usages in potentially non-popular repos. Trough cs.github.com I can find for example these usages:
* https://cs.github.com/wking/ci-tools/blob/64bde4a5f06d600725ddcc59f8d88410ec4fbe35/cmd/pod-scaler/admission.go?q=StartStandalone+language%3AGo#L39
* https://cs.github.com/openshift/cluster-api/blob/1101b984bc6d4e0c81dcb6d6e4651dd6dfbdbec8/exp/runtime/server/server.go?q=StartStandalone+language%3AGo#L239
* https://cs.github.com/varshavaradarajan/webhook-cr-poc/blob/7da2130be1d295be447196b77d5f4fbebe3af652/cmd/webhook-cr-poc/main.go?q=StartStandalone+language%3AGo#L40

So it does seem to be used.

> but really lots of things from the manager are then required (like scheme, or client).

Both of these can be constructed just fine without a manager.

Reply:
reverted, thanks!

Reply:
You commented that you reverted this but its still being removed?

Reply:
The `StartStandalone` part isn't necessary now, the scheme is neither used or injected, `webhook.Start` should work here